### ISSUE TITLE
After rolling upgrade, containers from original service don't get deleted if global scheduler label exist

### ISSUE THREAD

**Issue by aruneli** [rancher/rancher#1878]
With label io.rancher.scheduler.global set to true or false, rolling upgrade is not working as expected. New service gets created, but containers from old service don't get deleted. Here is the compose files I had used.

docker-compose.yml (original service)
```
db:
  restart: always
  environment:
    MYSQL_ROOT_PASSWORD: example
  labels:
    io.rancher.scheduler.global: 'true'
  tty: true
  image: mariadb
  stdin_open: true
wpalias:
  image: rancher/dns-service
  links:
  - wordpress:wordpress
wordpress:
  ports:
  - 8080:80/tcp
  restart: always
  labels:
    io.rancher.scheduler.global: 'true'
  tty: true
  image: wordpress
  links:
  - db:mysql
  stdin_open: true
DNSClient:
  restart: always
  tty: true
  image: sangeetha/testclient
  links:
  - 'wpalias:'
  stdin_open: true
```
rancher-compose.yml (original service)
```
db:
  scale: 1
wordpress:
  scale: 1
DNSClient:
  scale: 1
```
docker-compose.yml (used for upgrade)

```
wordpress:
wordpressnew:
  ports:
  - 8081:80/tcp
  restart: always
  labels:
    io.rancher.scheduler.global: 'true'
  tty: true
  image: wordpress:4.2
  stdin_open: true
  links:
  - db:mysql
wpalias:
  image: rancher/dns-service
  links:
  - wordpressnew:wordpress
DNSClient:
  restart: always
  tty: true
  image: sangeetha/testclient
  links:
  - 'wpalias:'
  stdin_open: true

```

**Comment by ibuildthecloud** [comment]
We currently don't support rolling upgrade for global.  We should add check and fail the CLI if global is enabled.

**Comment by ibuildthecloud** [comment]
@aruneli Fixed the issue in which we create the target service even if the source service has the global label.  The fix is in v0.3.0-rc2

**Comment by aruneli** [comment]
Verified fix in v0.3.0-rc2
